### DISCUSSION TITLE
Will not be able to use updater for official instance due to strict rules

### DISCUSSION THREAD

**Discussion by kernelcrunch** [tmiland/invidious-updater#75] [Ideas]
One of the rules to be listed as an official Invidious instance is that any modified source code include a link on the homepage/footer to the modified code.  I'm using the updater and it is working great.  However, it shows up as modified source code (with no link) and I can't see a way where I can update that link back to the source.  Would it make better sense to make a feature update to Invidious-Updater so that it complies with that requirement.  Or, would it be better if I fork my own, only to add that link?  (The former makes more sense to me but just asking)

Here is the related discussion: https://github.com/iv-org/documentation/issues/148

Or, do you think that if I'm running an official instance I should use the official source code?.  I like the updater, but I'll start over with the original source if necessary.  Thanks for the great work.

**Comment by tmiland** [comment]
Hello and thank you.

the script is using the official source code. No modifications are made. 

However, if you need to, you can change the [repo name](https://github.com/tmiland/Invidious-Updater/blob/3f0edd123918e0ab22bc5bad3cfe91bc6ad6d407/invidious_update.sh#L80) to your own fork in the script.

**Reply by kernelcrunch** [reply]
Thanks for the prompt reply.  That's what I originally thought, that the code was unmodified.  Thanks for confirming.  I've since figured out that it's simply the addition of the "modified_source_code_url" line in the config.yml that makes the footer display the modified source code link.
![image](https://user-images.githubusercontent.com/46076018/140499046-e868e5cd-8301-4290-aa90-7fe542083509.png)
If I take that line out, it just has the link to the source code.
![image](https://user-images.githubusercontent.com/46076018/140501580-1d9577fb-b90c-4b44-a27e-0db87fe8c069.png)
